docs/Video.md

```diff
@@ -0,0 +1,123 @@
+# `<Video>`
+
+FaCC that creates a `<video>` element to play video media, re-renders on video state changes.
+
+## Usage
+
+```jsx
+import {Video} from 'libreact/lib/Video';
+
+<Video autoPlay src='http://dailym.ai/2rG7TBS'>{(actions, state) => {
+  JSON.stringify(state, null, 4)
+}}</Video>
+```
+
+
+## Props
+
+In addition to props below also accepts all [React's media events](https://reactjs.org/docs/events.html#media-events).
+
+```tsx
+interface IVideoProps {
+  src: string;
+  autoPlay?: boolean;
+  loop?: boolean;
+  muted?: boolean;
+  preload?: 'none' | 'metadata' | 'auto';
+  volume?: number;
+  noJs?: React.ReactElement<any>;
+}
+```
+
+  - `src` - required, string, video source file URL.
+  - `autoPlay` - optional, boolean, whether to autoplay media, defaults to `false`.
+  - `loop` - optional, boolean, whether to repeat the media when it ends, defaults to `false`.
+  - `muted` - optional, boolean, whether to mute the media, defaults to `false`.
+  - `preload` - optional, string, `<video>` element preload attribute.
+  - `volume` - optional, number, media volume in `[0..1]` range, defaults to `1`.
+  - `noJs` - optional, React element(s) to render insided the `<video>` tag.
```

Comment:
Please check the syntax of this sentence.

Reply:
Looks good to me.